### DISCUSSION TITLE
Targeting a specific PCIe device?

### DISCUSSION THREAD

**Discussion by Morttase** [chipsec/chipsec#1308] [Q&A] [answered]
Good day to all, first off, I would like to thank the developers for this interesting project.

I stumbled across this framework while reading up on PCIe and I am unsure if I have missed out any information from the manual or the `--help` function for the pcie_fuzz module.

I would like to ask if there was any way to specifically target a single PCIe device connected to my test workstation for fuzzing purposes? Am I missing any additional arguments to instruct the script to target a single PCIe device?

Appreciate any guidance and help! Thank you!

**Comment by npmitche** [comment, marked as answer]
Hi @Morttase 

Sorry, we're slowly working on updating our documentation.

Current implementation allows you to pass in a BDF: 
`python chipsec_main.py -m tools.vmm.pcie_fuzz -a <B> <D> <F>`
Example:
`python chipsec_main.py -m tools.vmm.pcie_fuzz -a 0 2 f` 

Note BDF are interpreted as hex values.

**Reply by Morttase** [reply]
Hi @npmitche ,

Big thanks for the clear and concise guidance, much appreciated!
